pinot-common/src/main/java/org/apache/pinot/common/segment/SegmentMetadata.java

```diff
@@ -73,6 +73,23 @@
    */
   long getRefreshTime();
 
+  // methods specific to realtime segments
```

Comment:
I would prefer adding this comment to each of the comment block below. Otherwise, it seems as if the comment is valid for all methods below this line.

Reply:
Removed the comment as each method already indicates the same.